modules/ROOT/pages/a11ychecker.adoc

```diff
@@ -220,10 +222,32 @@ xref:a11ychecker_html_version[HTML version]:: HTML4 and HTML5
 
 WCAG 2.1 specification:: https://www.w3.org/WAI/WCAG21/Techniques/general/G95.html[G95 - Providing short text alternatives that provide a brief description of the non-text content].
 
+include::partial$misc/admon-acessibility-rule-i3-can-also-be-applied.adoc[]
+
+[[I3]]
+=== I3 - Image `+alt+` text is not greater than 100 characters
+
+*Rule description:* this rule checks that the `+alt+` attribute text of the image is not more than 100 characters.
+
+An _Image alternative text should be less than 100 characters_ warning dialog presents if the alternative (`+alt+`) text is longer than 100 characters. This dialog also presents the alternative text in an editable field, for immediate repair.
```

Comment:
I'm not sure if I'd say it's a "warning dialog". Isn't it just some text that appears in a dialog that is _already_ on the screen? I haven't played with it, though, so I'm not 100% sure.

@shanmen-tiny , thoughts?

Reply:
Given that we are including the partial above that has almost exactly the same information, won't this look duplicated also?

Reply:
> I'm not sure if I'd say it's a "warning dialog". Isn't it just some text that appears in a dialog that is _already_ on the screen? I haven't played with it, though, so I'm not 100% sure.
> 
> @shanmen-tiny , thoughts?

Yep, that's right. The dialog is already on the screen. I think the "warning dialog" here means the rule displays a `warning` notification level type of dialog?